src/posts/xarray-biology/index.md

```diff
@@ -0,0 +1,135 @@
+
+### Technical Barriers
+
+Once a potential user is convinced of `Xarray`'s value, they may still face technical barriers. Ranging from rough edges to missing features, however, none are insurmountable. An example of a rough edge is that, as of May 2025, you cannot use integers as keys in a `DataTree`. That is a problem, as integers are a natural key to use when tracking single cell lineages. Rough edges like this one haven’t been smoothed over yet because there has not been a user base of biologists using `Xarray`, discovering them, and raising issues to get them fixed.
+
+Until very recently, a missing feature was allowing `Xarray` coordinates to be more flexible than a fully instantiated array. Allowing for analytical transforms and having more flexible coordinates is critical to supporting Volumetric imaging applications. Excitingly, in February 2025, initial support for this was [merged](https://github.com/pydata/xarray/pull/9543)!
+
+Another area of potential improvement is support for sparse arrays. Many biology datasets are quite sparse, so robust support is a key feature. Xarray does have support for sparse arrays, but there are still open issues, for example [Issue 3212](https://github.com/pydata/xarray/issues/3213).
+
+### Data Loading/Lack of Integration
+
+Finally, we have been limited by a lack of integration with existing software tools. First, in loading the outputs of other tools into `Xarray` (do they have a `to_xarray` method?), and second, in other tools accepting `Xarray` arrays and using the extra features. For example, [Napari](https://napari.org/stable/) has had a long-standing [open issue](https://github.com/napari/napari/issues/14) about using `Xarray` to add extra information to dimension sliders.
+
+## Call To Action
+
+`Xarray` has fantastic potential to accelerate our ability to perform and share biological analyses and accelerate discovery. To realize this potential, there are a few things we as a community need to do.
+
+The first thing is to consider what we mean by “community”. There are two separate communities with some overlap: `Xarray` developers and biologists interested in using `Xarray`.
+
+### What can Xarray Do
+
+As a community of contributors to `Xarray`, we need to:
+
+- **Support**: Be responsive to technical limitations and bugs in Xarray that affect biologists
+- **Educate**: cross-post about improvements in Xarray to spaces with biologists
+
+### What can Biologists Do?
+
+**Try** using `Xarray` for your data. Find the smallest, most straightforward way to get your data into Xarray and start using it.
+
+Make sure to **ask for help** when you need it. To do so:
+
+- Post on forums (e.g., [image.sc](https://forum.image.sc/), or the [Xarray Github Discussions](https://github.com/pydata/xarray/discussions))
+- Join the new Xarray for Biology office hours [TODO LINK]
+- Book one on one time with Ian [TODO LINK]
```

Comment:
Need to create a link for these. Not quite sure best practice for this. github discussion?

Reply:
Use calendly for now IMO.